opm/autodiff/DuneMatrix.hpp

```diff
@@ -91,7 +91,11 @@ namespace Opm
             nnz = ia[rows];
 
 #if DUNE_VERSION_NEWER(DUNE_ISTL, 2, 3)
+#if DUNE_VERSION_NEWER(DUNE_ISTL, 2, 5)
```

Comment:
Does Dune work with different versions across modules? I would assume not, which should mean that DUNE_VERSION should have been a single thing, not one thing per module. Although perhaps that only applies to the core modules, and the facility is intended to work for user modules, too?

In any case, testing for common or istl should be equivalent, and if they are not that must be a bug!